pkg/cmd/cli/install/install.go

```diff
@@ -231,3 +251,41 @@ func (o *InstallOptions) Validate(c *cobra.Command, args []string, f client.Fact
 
 	return nil
 }
+
+// parseResourceRequests takes a set of CPU and memory requests and limit string
+// values and returns a ResourceRequirements struct to be used in a Container.
+// An error is returned if we cannot parse the request/limit.
+func parseResourceRequests(cpuRequest, memRequest, cpuLimit, memLimit string) (corev1.ResourceRequirements, error) {
+	var resources corev1.ResourceRequirements
+
+	parsedCPURequest, err := resource.ParseQuantity(cpuRequest)
+	if err != nil {
+		return resources, errors.WithMessage(err, "couldn't parse CPU request")
```

Comment:
let's switch these to use `errors.Wrap(...)` instead of `errrors.WithMessage(...)` so we get a stack trace as well.

Reply:
FYI, I changed this but I'm not seeing a stack trace printed with the error

Reply:
I guess we're probably just outputting it to stderr rather than using logrus so we don't get location output, which is fine here. In any case, we pretty much exclusively use `WithStack` or `Wrap` across the code base so no reason to deviate.